test/config/cucumber.yml

```diff
@@ -24,13 +24,11 @@ chrome: BROWSER=chrome
 ie: BROWSER=ie
 safari: BROWSER=safari
 
-################################################################################
-# Listed environments (default is set in config.yml)
-t: TEST_ENV=test -t @t,@test
-test: TEST_ENV=test -t @t,@test
+local: TEST_ENV=local -t @p,@prod
 
-u: TEST_ENV=acceptatie -t @u,@uat
-uat: TEST_ENV=acceptatie -t @u,@uat
+remote: SELENIUM_ENV=remote -t ~@ignore_on_remote TEST_ENV=remote
 
-p: TEST_ENV=production -t @p,@prod
-production: TEST_ENV=production -t @p,@prod
+################################################################################
+# Calliope.pro settings
+calliope_key: API_KEY=NDM4NmY5Njk5N2U2ZGJmNmRhYmI1ZDU1YjRkNDM1NzMxYjhlMDk2MjNlOGE5MWUyOTg2ZWUyMDJiZTVlMjk2YmJh
```

Comment:
Lets add the calliope key only in gitlab.

Reply:
Removed key